brave/brave-core
# Retain publisher status data on migration to publisher list v4

Resolves https://github.com/brave/brave-browser/issues/10840

- Modified migration logic so that `server_publisher_info` is retained for verified publishers that occur in the `publisher_info` table.
- Disabled purging of expired `server_publisher_info` records on prefix list update.
- Updated database test along with modifications to the 27 test database files.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Install 1.11.x (current release channel version), enable Rewards, confirm entire publisher list is downloaded.
- Visit some sites (mix of verified and unverified) to populate your AC table.
- Upgrade to latest.
- Navigate to `brave://rewards`.
  - Verify that verified checkmark appears for all verified publisher icons.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
